server.js

```diff
@@ -125,24 +125,24 @@ app.put('/api/v1/projects/:id', async (request, response) => {
 app.put('/api/v1/palettes/:id', async (request, response) => {
   const palette = request.body;
 
-  for (let requiredParameter of ['name', 'color1', 'color2', 'color3', 'color4', 'color5', 'project_id']) {
-      if (!palette[requiredParameter]) {
-        return response
-          .status(422)
-          .send({ error: `Expected format: { title: <String>, author: <String> }. You're missing a "${requiredParameter}" property.` });
-      }
-    }
+  if (!Object.keys(palette).length) {
```

Comment:
So this check to make sure each key has a length greater then 0.

Reply:
It checks that body object has any key-value pair for update.